lib/wakes/configuration.rb

```diff
@@ -16,12 +16,14 @@ def self.configure
   class Configuration
     attr_accessor :enabled
     attr_accessor :ga_profiles
+    attr_accessor :domains
```

Comment:
@chrismaddocks 

1. Can we use the term `hosts` instead of `domains`, in keeping with the field on `Wakes::Location`?
2. Can we make it more specific? This feels like the option you use to enable wakes to work with certain domains. What does that even mean? What about using `hosts_to_redirect`?